<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Items</class>
 <widget class="QDialog" name="Items">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="InventoryComboBox" name="type_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="InventoryLineEdit" name="number_lineEdit">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Operator:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="InventoryComboBox" name="operator_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>IP address:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="InventoryLineEdit" name="ipaddress_lineEdit">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="InventoryComboBox" name="location_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Manufacturer:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="InventoryLineEdit" name="manufacturer_lineEdit">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="InventoryLineEdit" name="model_lineEdit">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Serial No:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="5">
    <widget class="InventoryLineEdit" name="serialno_lineEdit">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Note:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="5">
    <widget class="QTextEdit" name="note_textEdit"/>
   </item>
   <item row="9" column="5" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Properties:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="InventoryComboBox" name="status_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="2" colspan="4">
    <widget class="QTreeView" name="properties_treeView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InventoryComboBox</class>
   <extends>QComboBox</extends>
   <header>inventorycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>InventoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>inventorylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Items</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Items</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
